src/net.h

```diff
@@ -1287,8 +1288,11 @@ struct NodeEvictionCandidate
     uint64_t nKeyedNetGroup;
     bool prefer_evict;
     bool m_is_local;
+    bool m_is_onion;
```

Comment:
Maybe called it `m_is_onion_service` to dissociate from the manually configured hidden services you mentioned above?

Reply:
Thanks @ariard. The latter ones would be `m_is_local`. I think I prefer to go with `m_is_onion` for `CNode::m_inbound_onion` and following the naming style of `m_is_local`.